dev/Expander/Expander.vcxitems

```diff
@@ -24,11 +24,6 @@
     <ClCompile Include="$(MSBuildThisFileDirectory)ExpanderAutomationPeer.cpp" />
   </ItemGroup>
   <ItemGroup>
-    <Page Include="$(MSBuildThisFileDirectory)Expander.xaml">
```

Comment:
The convention is to have the template in the `Expander.xaml`, with resources such as brushes in `Expander_themeresources.xaml`. I would revert this change.